bindings/wasm/examples-account/src/unchecked.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { AccountBuilder, Timestamp, ExplorerUrl, Document, Storage } from './../../node/identity_wasm.js';
+
+/**
+ * This example demonstrates how to update the custom properties of a DID document directly 
+ * and publish it without performing validation.
+ */
+async function unchecked(storage?: Storage) {
+
+    // The creation step generates a keypair, builds an identity
+    // and publishes it to the IOTA mainnet.
+    let builder = new AccountBuilder({
+        storage,
+    });
+    let account = await builder.createIdentity();
+
+    // Get a copy of the document this account manages.
+    // We will apply updates to the document, and overwrite the account's current document.
+    let document = account.document();
+
+    // Print the local state of the DID Document
+    console.log(`[Example] Document before update`, account.document());
+
+    // Add a custom property to the document.
+    const documentJSON = document.toJSON();
+    documentJSON["doc"]["myCustomPropertyKey"] = "value";
+    document = Document.fromJSON(documentJSON);
```

Comment:
This should use the new `setPropertyUnchecked` function.

```suggestion
    // Add a custom property to the document.
    docmuent.setPropertyUnchecked("myCustomPropertyKey", "value");
```

Reply:
Done in #690.